minutes/MeetingNotes - 17 May 2022.md

```diff
@@ -0,0 +1,34 @@
+# Notes from today meeting May 17
+
+## Housekeeping
+- Please use our repo https://github.com/opiproject/opi-prov-life
+- Please use our slack https://opi-project.slack.com/archives/C0342L6T7EC
+- meeting will be shortened to 45 minutes
+
+## Docs
+- please look at https://opentelemetry.io/docs/
+- please look at https://github.com/silicom-ltd/uBMC
+- please look at https://opencomputeproject.github.io/onie/overview/
+- please look at https://fidoalliance.org/intro-to-fido-device-onboard/
+- please look at https://www.rfc-editor.org/rfc/pdfrfc/rfc8572.txt.pdf (Secure Zero Touch Provisioning (SZTP))
+
+## Monitoring & Telemetry
+- we adoped https://opentelemetry.io/ for OPI
```

Comment:
I'm curious about this - are there any specific use cases that adopting a specific toolkit addresses?

Reply:
thanks for the feedback @shachartal 

OTEL is a SPEC, every customer/user/developer can use whatever SDK and Collector they choose as long it's in the OTEL specification. one can use `OpenTelemetry Collector` or any other implementation of OTEL of their choosing

The idea is to have a common protocol that all DPU vendors use for metering, racing, logging instead of trying to adopt many custom protocols... Integrator companies really struggling handle all those different protocols and prefer to have a single protocol. OTEL was a choice we made. and we have flashed out pros and cons. This is the second most contributed project in CNCF today after k8s.

The reference architecture we working on recommends to run OTEL collector on the DPU
So vendor that want to do proprietary metering and tracing can do it, but should convert to OTEL inside the DPU